Kconfig

```diff
@@ -1591,7 +1591,9 @@ menu "LVGL configuration"
 			int "Font manager name max length"
 			depends on LV_USE_FONT_MANAGER
 			default 32
-
+		config LV_USE_XML
+			bool "Enable loading XML UIs runtime"
+			default n
```

Comment:
Minor point.

Default n is not necessary, Kconfig by standard set the option to n when nothing is specified.

Reply:
Added.

Reply:
Great feel free to resolve here.